app/assets/javascripts/discourse/app/templates/user.hbs

```diff
@@ -384,7 +384,7 @@
                       @model={{this.model.invited_by}}
                     >{{this.model.invited_by.username}}</LinkTo></dd></div>
               {{/if}}
-              {{#if this.model.trust_level}}
+              {{#if this.hasTrustLevel}}
```

Comment:
thought: it's a real pity Ember Templates doesn't have the handlebars built-in helper to include `0` as truthy (https://handlebarsjs.com/guide/builtin-helpers.html#includezero), would have made this a simpler change.